2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_setup.py:_flush():71] setting env: {}
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220304_121256-32qzez60/logs/debug.log
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220304_121256-32qzez60/logs/debug-internal.log
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_init.py:init():404] calling init triggers
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-04 12:12:56,418 INFO    MainThread:670822 [wandb_init.py:init():460] starting backend
2022-03-04 12:12:56,418 INFO    MainThread:670822 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-04 12:12:56,424 INFO    MainThread:670822 [backend.py:ensure_launched():216] starting backend process...
2022-03-04 12:12:56,429 INFO    MainThread:670822 [backend.py:ensure_launched():222] started backend process with pid: 670831
2022-03-04 12:12:56,429 INFO    MainThread:670822 [wandb_init.py:init():469] backend started and connected
2022-03-04 12:12:56,431 INFO    MainThread:670822 [wandb_init.py:init():533] updated telemetry
2022-03-04 12:12:56,443 INFO    MainThread:670822 [wandb_init.py:init():563] communicating current version
2022-03-04 12:12:56,883 INFO    MainThread:670822 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-04 12:12:56,883 INFO    MainThread:670822 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-04 12:12:57,161 INFO    MainThread:670822 [wandb_init.py:init():606] starting run threads in backend
2022-03-04 12:12:59,696 INFO    MainThread:670822 [wandb_run.py:_console_start():1810] atexit reg
2022-03-04 12:12:59,696 INFO    MainThread:670822 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-04 12:12:59,696 INFO    MainThread:670822 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-04 12:12:59,698 INFO    MainThread:670822 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-04 12:12:59,699 INFO    MainThread:670822 [wandb_init.py:init():633] run started, returning control to user process
2022-03-04 12:13:13,596 INFO    MainThread:670822 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-04 12:13:13,596 INFO    MainThread:670822 [wandb_run.py:_restore():1752] restore
2022-03-04 12:13:16,226 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-04 12:13:16,555 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-04 12:13:17,177 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 6880
}

2022-03-04 12:13:17,278 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16479
}

2022-03-04 12:13:17,381 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16479
}

2022-03-04 12:13:17,482 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 16479
}

2022-03-04 12:13:17,585 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:17,687 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:17,790 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:17,892 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:17,994 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:18,096 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:18,419 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}

2022-03-04 12:13:19,064 INFO    MainThread:670822 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16479
  total_bytes: 16479
}
local_info {
}

2022-03-04 12:13:20,127 INFO    MainThread:670822 [wandb_run.py:_append_history():2130] rendering history
2022-03-04 12:13:20,128 INFO    MainThread:670822 [wandb_run.py:_append_summary():2085] rendering summary
2022-03-04 12:13:20,128 INFO    MainThread:670822 [wandb_run.py:_append_files():2180] logging synced files
